sencha/index.py

```diff
@@ -77,8 +77,17 @@ def load_nodegraph(*args, **kwargs):
         return khmer.Nodegraph.load(*args, **kwargs)
 
 
+def maybe_read_peptide_file(peptide_file):
+    records = []
+    try:
+        records = screed.open(peptide_file)
```

Comment:
should we log/record which files were included in the peptide_bloom filter?

Reply:
we are logging what didn't get included, so, that would give info about what were included, so I think we are good with logs here. 

unless you want more info, I would logger.debug